packages/perennial/contracts/README.md

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
these are still here

Reply:
do we need readme anymore? was more of a writeup / justification. also seems out of place here in general

Reply:
Actually good call, let's get read of that readme, but section 3 is good info to include in the `@dev` portion of the contract natspec (basically just explaining how the encoding works)